papis/commands/add.py

```diff
@@ -613,6 +613,11 @@ def cli(files: List[str],
                 "Select matching importers (for instance 0, 1, 3-10, a, all...)")
 
             matching_importers = [matching_importers[i] for i in matching_indices]
+        elif matching_importers:
+            if len(matching_importers) >= 2:
+                matching_importers = [matching_importers[1]]   
+            else:
+                matching_importers = [matching_importers[0]]
```

Comment:
Can you explain the logic behind this? I'd expect that in batch mode, we just merge all matching importers.